### PR TITLE
Chiyavan

### PR DESCRIPTION
**Pull request by cbanavi**
Week 1 Tasks

### CODE REVIEW

**Review by jenneft — CHANGES_REQUESTED** [review]
Great start! Let's enhance your code a bit ;) standing by for the code lines.

File: src/chiyavan/divisionWithoutOperator_Chiyavan.java
Comment (jenneft) [inline src/chiyavan/divisionWithoutOperator_Chiyavan.java:1]: +1

File: src/chiyavan/finra_Chiyavan.java
Comment (jenneft) [inline src/chiyavan/finra_Chiyavan.java:1]: +1

File: src/chiyavan/oddOrEven_Chiyavan.java
Comment (jenneft) [inline src/chiyavan/oddOrEven_Chiyavan.java:1]: +1

**Review by jenneft — APPROVED** [review]
Merge away :)

File: src/chiyavan/divisionWithoutOperator_Chiyavan.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package chiyavan;
+
+public class divisionWithoutOperator_Chiyavan {
```
[the remark this answers is not in the document]
Comment (cbanavi) [reply src/chiyavan/divisionWithoutOperator_Chiyavan.java]: Just updated

**Comment by PendarShams** [comment]
LGTM!

**Pull request merged**